skyportal/handlers/api/comment.py

```diff
@@ -26,7 +27,7 @@ def users_mentioned(text):
 
 class CommentHandler(BaseHandler):
     @auth_or_token
-    def get(self, comment_id):
+    def get(self, comment_id, comment_on_what=None):
```

Comment:
Agree with Ari that it would be simpler to just set the default to `"object"` here and not have the `if comment_on_what is None:` line at the beginning of the handler.